docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+services:
+  integration_tests:
+    build:
+      dockerfile: Dockerfile
+      context: ${GSY_E_TARGET_REPO:-./}
+    container_name: ${CONTAINER_NAME:-gsy-e-tests}
+    entrypoint: ["gsy-e"]
+    command: ${COMMAND:-}
+    environment:
+      - REDIS_URL=${REDIS_URL:-redis://redis.container:6379/}
+    volumes:
+      - ${GSY_E_TARGET_REPO:-integration_tests}:/gsy-e
```

Comment:
Why is the default volume `integration_tests`? Shouldn't it be the complete gsy-e folder? That's what `GSY_E_TARGET_REPO` would target.

Reply:
The default volume is `integration_tests` for it to be possible to execute the tests without necessarily specifying GSY_E_TARGET_REPO variable. The data stored on integration_tests volume would correspond to the same data which is copied during the image build.

Therefore, if GSY_E_TARGET_REPO is not specified, the code under which the tests run is the image's default code, (most likely the master remote branch depending on how the image was built).

As you mention, if GSY_E_TARGET_REPO is specified, then the code under gsy-e folder of the target repo is the one that will be used for the tests execution.

Reply:
Ah sorry, I thought that `integratino_tests` here was the folder name, but it's actually the name of the volume. In that case, as I've mentioned in another comment, I'd just rename that volume to be `gsy-e` for clarity.